tests/framework/Optimizers/Persistence/parabola.py

```diff
@@ -0,0 +1,25 @@
+#***************************************
+#* Simple analytic test ExternalModule *
+#***************************************
+#
+# Computes an inverse parabola with maximum value at x=0 of 1.0.
+#
+import numpy as np
+
+#static seed
+np.random.seed(42)
+
+def base(values):
+  return -sum(x**2 for x in values)+1.
```

Comment:
I made a minor change to this line as the online editor led me to believe the variable `l` was actually a `1`.